packages/meditrak-app/app/authentication/actions.js

```diff
@@ -49,7 +49,7 @@ export const login = (emailAddress, password) => async (
   const loginCredentials = {
     emailAddress,
     password,
-    deviceName: DeviceInfo.getDeviceName(),
+    deviceName: await DeviceInfo.getDeviceName(),
```

Comment:
I guess this is why it comes back as unknown for you before?

Reply:
No the `await` was in place when I reported that. Still not sure, I think it's just the Genymotion device I have set up doesn't give a name.